src/repository/cart.repository.ts

```diff
@@ -0,0 +1,24 @@
+import mongoose, { Schema } from "mongoose";
+import {CartEntity, CartItemEntity} from "../model/cart.model.js"
+
+export const cartItemsSchema: Schema = new Schema({
```

Comment:
I wouldn't mix repositories and schemas technically you can have multiple repository implementations. It's probably better to have a separate place dedicated to schema definitions.